lib/challenge.js

```diff
@@ -20,6 +20,14 @@
 
 // Answer Starts Here
 
+var alarm clock = {
+    display: 'digital',
+    color: 'black',
+    weight: '8 ounces',
+    togglePowerSource: ['AC', 'battery']
```

Comment:
You need commas at the end of this line and at the end of the line that follows it. Commas separate key-value pairs; the array after the colon is the value segment of this key-value pair.